src/java/org/apache/cassandra/schema/CompressionParams.java

```diff
@@ -505,7 +651,6 @@ static class Serializer implements IVersionedSerializer<CompressionParams>
     {
         public void serialize(CompressionParams parameters, DataOutputPlus out, int version) throws IOException
         {
-            assert version >= MessagingService.VERSION_40;
```

Comment:
why remove these ?

Reply:
where would we ?   not in an online cluster.

Reply:
wow I don't remember this at all ... feel free to put it back